backend/main.go

```diff
@@ -187,17 +188,79 @@ func initialize(servicePort, databaseURL, redisURL, loggingURL, statsdURL, certP
 		r.Mount("/locations", locations.NewLocationsRouter(&locations.LocationsRouter{Svcs: svc}).Router)
 	})
 
+	r.Group(func(r chi.Router) {
```

Comment:
We should clean this up to get all of this logic out of the main pkg. Start with this (move to an `endpoints` pkg?) and we'll have another PR cleaning up the rest.

Reply:
I was planning on refactoring some of this code to be a cleaner architecture later this week, including cleaning up main. No reason routes, logic, and middleware should exist in main anyway. So you should see a PR with the appropriate changes soon